### DISCUSSION TITLE
Welcome to shadcn-rails Discussions!

### DISCUSSION THREAD

**Discussion by aviflombaum** [aviflombaum/shadcn-rails#72] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kmooventhan98** [comment]
Why [shadcn-rails](https://github.com/aviflombaum/shadcn-rails) doesn't have much component comparing to this https://ui.shadcn.com/docs/components/accordion

**Reply by aviflombaum** [reply]
Hi!

We do https://shadcn.rails-components.com/docs/components/accordion

I'm also trying to find time to continue this project - I have big plans for this gem but ya https://x.com/aviflombaum/status/1867235746928803929

Hope this helps!

**Reply by kmooventhan98** [reply]
cool yay 🎉
